configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -311,7 +309,6 @@ public FeatureFlags(FlagSource source, ApplicationId appId, Version version) {
         }
         @Override public boolean alwaysMarkPhraseExpensive() { return alwaysMarkPhraseExpensive; }
         @Override public int contentLayerMetadataFeatureLevel() { return contentLayerMetadataFeatureLevel; }
-        @Override public boolean dynamicHeapSize() { return dynamicHeapSize; }
```

Comment:
Keep this and the the flag with updated default.
Then GC flag override this version is rolled out.
Then you can remove this and the flag definition afterwards

Reply:
The interface's default implementation defines the new default value. There is no need for keeping the wiring anymore.

Reply:
Yes, that is correct. But you now leave a flagdefinition file that has no effect, other that than preventing an error messeage. That is why I like to keep the wiring one version longer and remove flag def and wiring in same PR.